### PR TITLE
ahoy 2.0.1

### PR DESCRIPTION
**Pull request by ocean**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/ahoy.rb [hunk 1]
```diff
@@ -20,13 +20,7 @@ class Ahoy < Formula
...
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    bin_path = buildpath/"src/github.com/ahoy-cli/ahoy"
-    bin_path.install Dir["*"]
-    cd bin_path do
-      system "go", "build", "-o", bin/"ahoy", "-ldflags", "-X main.version=#{version}", "."
-    end
+    system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}-homebrew", "-o", bin/"ahoy", "."
```
Comment (chenrui333) [inline Formula/ahoy.rb:23]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w X main.version=#{version}-homebrew")
```

File: Formula/ahoy.rb [hunk 2]
```diff
@@ -52,5 +46,7 @@ def caveats
           cmd: echo "Hello Homebrew!"
     EOS
     assert_equal "Hello Homebrew!\n", `#{bin}/ahoy hello`
+
+    assert_equal version.to_s + "-homebrew", shell_output("#{bin}/ahoy --version").strip
```
Comment (chenrui333) [inline Formula/ahoy.rb:50]:
```suggestion
    assert_equal "#{version}-homebrew", shell_output("#{bin}/ahoy --version").strip
```

File: Formula/ahoy.rb (hunk 1 above)
Comment (ocean) [reply Formula/ahoy.rb:23]: Thanks for this suggestion @chenrui333 - I played around with the `std_go_args()` but couldn't get them to put my version string in properly, but I'll try again now and hopefully can get it working.

File: Formula/ahoy.rb (hunk 2 above)
Comment (ocean) [reply Formula/ahoy.rb]: Ok, updated this one too 👍

File: Formula/ahoy.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/ahoy.rb]: interesting

Comment (chenrui333) [reply Formula/ahoy.rb]: it builds/tests out fine in my local.

Comment (ocean) [reply Formula/ahoy.rb]: Got this working now, thanks 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**